### PR TITLE
[FLINK-24121][flink-python] There are spelling mistakes in the notes.

### PR DESCRIPTION
**Pull request by q977734161**
## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency):  no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers:  no
  - The runtime per-record code paths (performance sensitive): no 

## Documentation

  - This pull request does not introduce a new feature.

### CODE REVIEW

**Review by infoverload — APPROVED** [review]
lgtm

**Review by RyanSkraba — APPROVED** [review]
Nice cleanup!

**Comment by sjwiesman** [comment]
Merging...

File: flink-python/dev/lint-python.sh [hunk 1]
```diff
@@ -460,7 +460,7 @@ function create_dir() {
 # Set created py-env in $PATH for tox's creating virtual env
 function activate () {
     if [ ! -d $CURRENT_DIR/.conda/envs ]; then
-        echo "For some unkown reasons,missing the directory $CURRENT_DIR/.conda/envs,\
+        echo "For some unknown reasons,missing the directory $CURRENT_DIR/.conda/envs,\
```
[the remark this answers is not in the document]
Comment (sjwiesman) [reply flink-python/dev/lint-python.sh:463]: I'll apply when merging

**Pull request closed without merging**